usaspending_api/awards/models.py

```diff
@@ -828,3 +828,96 @@ class Subaward(DataSourceTrackedModel):
     class Meta:
         managed = True
         unique_together = (('subaward_number', 'award'),)
+
+
+class SummaryAwardView(models.Model):
+
+    # Fields
+    action_date = models.DateField(blank=True, null=True)
```

Comment:
Some tables (like `transaction_normalized`) don't allow null `action_date` values. TBH I don't think it matters what this ORM model has, but I don't know for sure